tests/mocha/keydown_test.js

```diff
@@ -0,0 +1,277 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+suite('Key Down Tests', function() {
```

Comment:
nit: I would remove "Tests" from the title

Reply:
Done